app/routes/bdb/SortCompanies.ts

```diff
@@ -7,29 +7,43 @@ import type { TransformedAdminCompany } from 'app/reducers/companies';
 import type { ParsedQs } from 'qs';
 
 const sortByAttribute = (attribute) => (ascending) => (a, b) => {
-  if ((!a[attribute] && !b[attribute]) || a[attribute] === b[attribute]) {
+  const aAttr = a?.[attribute];
+  const bAttr = b?.[attribute];
```

Comment:
Not critical - but it doesn't hurt to avoid abbreviations where possible;
```suggestion
  const aAttribute = a?.[attribute];
  const bAttribute = b?.[attribute];
```

Reply:
Agreed. Changed it now 😄